lib/message_bus.rb

```diff
@@ -503,6 +503,18 @@ def last_id(channel, site_id = nil)
     backend_instance.last_id(encode_channel_name(channel, site_id))
   end
 
+  # Get the ID of the last message published on multiple channels
+  #
+  # @param [Array<String>] channels - array of channels to fetch
+  # @param [String] site_id - the ID of the site by which to filter
+  #
+  # @return [Array<Integer>] the channel-specific IDs of the last message published to each requested channel
```

Comment:
Isn't it returning a `Hash`?

```suggestion
  # @return [Hash] the channel-specific IDs of the last message published to each requested channel
```